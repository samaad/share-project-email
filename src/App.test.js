import React from "react";
import renderer from "react-test-renderer";
import { shallow } from "enzyme";
import App from "./App";

test("should render the App component", () => {
  const component = renderer.create(<App />);
  expect(component).toMatchSnapshot();
});

test("should render pannel", () => {
  const component = shallow(<App />);
  expect(component.text()).toEqual("<Panel />");
});
