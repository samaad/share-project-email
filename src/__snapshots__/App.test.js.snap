// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the App component 1`] = `
<div
  className="App"
>
  <div
    className="panel"
  >
    <div
      className="panel__body"
    >
      <div
        className="header"
      >
        <p
          dangerouslySetInnerHTML={
            Object {
              "__html": "Share&nbsp;<span class='header__projectname'>Project name</span>&nbsp;with others",
            }
          }
        />
      </div>
      <div
        className="email"
      >
        <div
          className="email__wrapper"
        >
          <input
            className="email__wrapper__inputtext"
            onChange={[Function]}
            onKeyDown={[Function]}
            placeholder="Type email addresses..."
            value=""
          />
        </div>
      </div>
    </div>
    <div
      className="panel__footer"
    >
      <div
        className="footer"
      >
        <button
          className="button"
        >
          Add email
        </button>
        <button
          className="button"
        >
          Get emails count
        </button>
      </div>
    </div>
  </div>
</div>
`;

exports[`should render the Panel component 1`] = `
<div
  className="App"
>
  <div
    className="panel"
  >
    <div
      className="panel__body"
    >
      <div
        className="header"
      >
        <p
          dangerouslySetInnerHTML={
            Object {
              "__html": "Share&nbsp;<span class='header__projectname'>Project name</span>&nbsp;with others",
            }
          }
        />
      </div>
      <div
        className="email"
      >
        <div
          className="email__wrapper"
        >
          <input
            className="email__wrapper__inputtext"
            onChange={[Function]}
            onKeyDown={[Function]}
            placeholder="Type email addresses..."
            value=""
          />
        </div>
      </div>
    </div>
    <div
      className="panel__footer"
    >
      <div
        className="footer"
      >
        <button
          className="button"
        >
          Add email
        </button>
        <button
          className="button"
        >
          Get emails count
        </button>
      </div>
    </div>
  </div>
</div>
`;
